modules/related-posts/jetpack-related-posts.php

```diff
@@ -92,23 +143,21 @@ public function __construct( $blog_id_local, $blog_id_wpcom ) {
 	/**
 	 * Add a checkbox field to Settings > Reading for enabling related posts.
 	 *
+	 * @access public
 	 * @action admin_init
 	 * @uses add_settings_field, __, register_setting, add_action
-	 * @return null
+	 * @return void
 	 */
 	public function action_admin_init() {
-
-		// Add the setting field [jetpack_relatedposts] and place it in Settings > Reading
+		// Add the setting field [jetpack_relatedposts] and place it in Settings > Reading.
 		add_settings_field( 'jetpack_relatedposts', '<span id="jetpack_relatedposts">' . __( 'Related posts', 'jetpack' ) . '</span>', array( $this, 'print_setting_html' ), 'reading' );
 		register_setting( 'reading', 'jetpack_relatedposts', array( $this, 'parse_options' ) );
-		add_action('admin_head', array( $this, 'print_setting_head' ) );
-
-		if( 'options-reading.php' == $GLOBALS['pagenow'] ) {
-			// Enqueue style for live preview on the reading settings page
+		add_action( 'admin_head', array( $this, 'print_setting_head' ) );
+		if ( 'options-reading.php' === $GLOBALS['pagenow'] ) {
```

Comment:
Changing `==` to `===` or `!=` to `!==` is not trivial. Please either 
a) leave all of them as before or 
b) write PHPUnit tests that guarantee that this won't break anything.
Also, WP PHP coding standards don't say anything about this. They even have examples using `==` like https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#ternary-operator and Yoda Condition below it.